lib/appoptics_apm/frameworks/rails/inst/action_controller.rb

```diff
@@ -88,17 +88,17 @@ def render_with_appoptics(*args, &blk)
   AppOpticsAPM.logger.info '[appoptics_apm/loading] Instrumenting actioncontroller' if AppOpticsAPM::Config[:verbose]
   require "appoptics_apm/frameworks/rails/inst/action_controller#{Rails::VERSION::MAJOR}"
   if Rails::VERSION::MAJOR >= 5
-    ::ActionController::Base.send(:prepend, ::AppOpticsAPM::Inst::ActionController)
+    ::ActionController::Base.send(:prepend, AppOpticsAPM::Inst::ActionController)
```

Comment:
Is `::` needed for `ActionController`? There is no `ActionController` module in this file.

Reply:
no, there are a few `::` that I missed here.
I doesn't matter if it is defined in this file, it matters if we are in a scope that could have a definition with the same name.